From 39c50b5b1e83f700adc0ffff0fdcd98549cb201e Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:54:10 -0400
Subject: [fs] ext4: fix a typo in extents.c

Message-id: <1414058125-4183-97-git-send-email-lczerner@redhat.com>
Patchwork-id: 98239
O-Subject: [RHEL 7.1 PATCH v2 096/171] ext4: fix a typo in extents.c
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Yongqiang Yang <xiaoqiangnk@gmail.com>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit 9e740568bc65a82845330b9439514255140b94aa

 Signed-off-by: Yongqiang Yang <yangyongqiang01@baidu.com>
 Reviewed-by: Carlos Maiolino <cmaiolino@redhat.com>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/extents.c b/fs/ext4/extents.c
index 87bd1b9..22256c5 100644
--- a/fs/ext4/extents.c
+++ b/fs/ext4/extents.c
@@ -3477,7 +3477,7 @@ static int ext4_ext_convert_to_initialized(handle_t *handle,
  WARN_ON(map->m_lblk < ee_block);
  /*
   * It is safe to convert extent to initialized via explicit
-  * zeroout only if extent is fully insde i_size or new_size.
+  * zeroout only if extent is fully inside i_size or new_size.
   */
  split_flag |= ee_block + ee_len <= eof_block ? EXT4_EXT_MAY_ZEROOUT : 0;
 
-- 
1.7.1